packages/telemetry/lib/node-telemetry.js

```diff
@@ -7,30 +7,30 @@ const { ATTR_SERVICE_NAME } = require('@opentelemetry/semantic-conventions')
 const { workerData } = require('node:worker_threads')
 const { resolve } = require('node:path')
 const { tmpdir } = require('node:os')
-const logger = require('abstract-logging')
 const { statSync, readFileSync } = require('node:fs') // We want to have all this synch
-const util = require('node:util')
-const debuglog = util.debuglog('@platformatic/telemetry')
+const { pino } = require('pino')
+const { debuglog } = require('node:util')
 const {
   ConsoleSpanExporter,
   BatchSpanProcessor,
   SimpleSpanProcessor,
-  InMemorySpanExporter,
+  InMemorySpanExporter
 } = require('@opentelemetry/sdk-trace-base')
 
 const { PgInstrumentation } = require('@opentelemetry/instrumentation-pg')
 const { HttpInstrumentation } = require('@opentelemetry/instrumentation-http')
-const {
-  UndiciInstrumentation,
-} = require('@opentelemetry/instrumentation-undici')
+const { UndiciInstrumentation } = require('@opentelemetry/instrumentation-undici')
 
 // See: https://www.npmjs.com/package/@opentelemetry/instrumentation-http
 // When this is fixed we should set this to 'http' and fixe the tests
 // https://github.com/open-telemetry/opentelemetry-js/issues/5103
 process.env.OTEL_SEMCONV_STABILITY_OPT_IN = 'http/dup'
 
-const setupNodeHTTPTelemetry = (opts) => {
+const logger = pino({ level: debuglog('@platformatic/telemetry').enabled ? 'debug' : 'silent' })
```

Comment:
Please revert this. If we are adding logging we would need to pass this through for other modules.